pkg/ping/build.go

```diff
@@ -32,14 +32,15 @@ func pingAttributes(info *version.Info, pods []string, minKubeletVersion *string
 	metrics := scraper.GetMetrics().Get()
 
 	cp := console.ClusterPing{
-		CurrentVersion: strings.TrimPrefix(vs[0], "v"),
-		KubeletVersion: minKubeletVersion,
-		Distro:         lo.ToPtr(findDistro(append(pods, info.GitVersion))),
-		HealthScore:    &hs,
-		NodeCount:      &nodCount,
-		PodCount:       podCount,
-		NamespaceCount: &namespaceCount,
-		NodeStatistics: ns,
+		CurrentVersion:   strings.TrimPrefix(vs[0], "v"),
+		KubeletVersion:   minKubeletVersion,
+		Distro:           lo.ToPtr(findDistro(append(pods, info.GitVersion))),
```

Comment:
don't believe this is setting distro to OPENSHIFT

Reply:
I missed it.

Reply:
OPENSHIFT is missing on the API site